va/va.go

```diff
@@ -984,6 +995,9 @@ func (va *ValidationAuthorityImpl) performRemoteValidation(ctx context.Context,
 				// instances.
 				result <- prob
 			} else {
+				// Otherwise the error was not an expected non-sucess problem result and
+				// represents an internal error. The real error has already been logged
```

Comment:
s/error/errors/.

Reply:
Why? This is inside of a loop through `range errors`. The `err` item being discussed is a singular error.

Reply:
The existing comment on the `result <- prob` case above this `} else {` also talks about an error in the singular form.